resources/env/delivery/bin/crafter-setenv.sh

```diff
@@ -110,6 +110,21 @@ export CRAFTER_ENCRYPTION_SALT=${CRAFTER_ENCRYPTION_SALT:="DgGN9xhq3GOn6zxg"}
 # -------------------- Configuration variables --------------------
 export CRAFTER_ENVIRONMENT=${CRAFTER_ENVIRONMENT:=default}
 
+# -------------------- Serverless variables --------------------
+export AWS_S3_ENDPOINT=${AWS_S3_ENDPOINT:=""}
+export AWS_S3_PATH_STYLE_ACCESS=${AWS_S3_PATH_STYLE_ACCESS:="false"}
+
+export SERVERLESS_NAMESPACE=${SERVERLESS_NAMESPACE:=cloud-sites}
+
+export PREVIEW_BUCKET_NAME_PATTERN=${SERVERLESS_NAMESPACE}-blobs-\${siteName}
+export PREVIEW_BUCKET_PREFIX_PATTERN=sandbox
```

Comment:
I don't thing we need to add a prefix for preview. Since it's a different bucket than the others, normally this is empty